src/openklant/components/klantinteracties/models/internetaken.py

```diff
@@ -71,3 +75,15 @@ class InterneTaak(models.Model):
     class Meta:
         verbose_name = _("interne taak")
         verbose_name_plural = _("interne taken")
+
+    def save(self, *args, **kwargs):
+        if not self.nummer:
+            max_nummer = (
+                int(InterneTaak.objects.aggregate(models.Max("nummer"))["nummer__max"])
+                or 0
+            )
+            self.nummer = str(max_nummer + 1).rjust(10, "0")
+            if len(self.nummer) > 10:
+                raise Conflict(_("Nummer mag maximaal 10 characters bevatten."))
```

Comment:
Kan je hiervan een utility functie maken? Je kan het model (self) en het attribute (nummer) als argumenten meegeven bijvoorbeeld.

Ook de melding mag anders: "Er kon niet automatisch een opvolgend nummer worden gegenereerd. Het maximaal aantal tekens is bereikt."